packaging/linux/docker/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# This Dockerfile results in a simple Docker image with
+# a statically compiled Keybase binary in /usr/bin/keybase.
+FROM golang:1.12.4-stretch AS builder
+LABEL maintainer="Keybase <admin@keybase.io>"
+
+ENV KEYBASE_SKIP_32_BIT=1
+
+COPY . /go/src/github.com/keybase/client
+RUN /go/src/github.com/keybase/client/packaging/linux/build_binaries.sh \
+    production /
+RUN chmod +x /binaries/amd64/usr/bin/keybase
+
+FROM debian:10
+
+RUN apt-get update \
+    && apt-get install -y gnupg2 \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV TINI_VERSION v0.18.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini.asc /tini.asc
+RUN gpg --batch --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 595E85A6B1B4779EA4DAAEC70B588DFF0527A9B7 \
+    && gpg --batch --verify /tini.asc /tini \
+    && chmod +x /tini
+
+RUN useradd --create-home --shell /bin/bash keybase
+USER keybase
```

Comment:
@jzila I remember there used to be some weirdness wrt mountpoints about using the user "keybase", is it still a problem?

Reply:
Not sure what you're referring to, we use `keybase` in the test dockers.

Reply:
The redirector does look for paths that contain "keybase/" when trying to find the right mountpoint for redirecting.  But it should only matter if there will be another mountpoint in the container that also contains "keybase".

Reply:
Shouldn't be an issue - I'll elaborate further in another reply here.